deeppavlov/dataset_readers/kbqa_reader.py

```diff
@@ -0,0 +1,34 @@
+from pathlib import Path
+
+from deeppavlov.core.common.registry import register
+from deeppavlov.core.data.dataset_reader import DatasetReader
+from deeppavlov.core.data.utils import download_decompress
+
+
+@register('kbqa_reader')
+class KBQAReader(DatasetReader):
+    """Class to read test set of questions and answers for knowledge base question answering"""
+
+    def read(self, data_path: str):
+        data_path = Path(data_path)
+        files = list(data_path.glob('*.txt'))
+        if 'test_set_with_answers.txt' not in {file_path.name for file_path in files}:
+            url = 'http://files.deeppavlov.ai/kbqa/test_set_with_answers.txt'
+            data_path.mkdir(exist_ok=True, parents=True)
+            download_decompress(url, data_path)
+            files = list(data_path.glob('*.txt'))
+        dataset = {}
+
+        dataset["test"] = self.parse_ner_file(files[0])
```

Comment:
Are you sure you want to use `files[0]` and not just `data_path / 'test_set_with_answers.txt'`?